src/main/java/edu/unc/lib/boxc/migration/cdm/services/SipService.java

```diff
@@ -130,20 +140,33 @@ public List<MigrationSip> generateSips(SipGenerationOptions options) {
                 String entryType = rs.getString(3);
 
                 WorkGenerator workGen = workGeneratorFactory.create(cdmId, cdmCreated, entryType);
+                // update progress bar
+                currentNumber++;
+                DisplayProgressUtil.displayProgress(currentNumber, total);
                 try {
                     workGen.generate();
                 } catch (SkipObjectException e) {
                     // Skipping
                 }
             }
+            DisplayProgressUtil.finishProgress();
+
+            // set up sips progress bar
+            long currentCount = 0;
```

Comment:
`currentNumber` and `currentCount` seem like they'd be pretty easy to get mixed up. How about `workCount` and `destinationCount`?